scripts/pre_push_hook.py

```diff
@@ -36,7 +36,8 @@
 import subprocess
 import sys
 
-import python_utils
+sys.path.append(os.getcwd())
+import python_utils  # isort:skip  # pylint: disable=wrong-import-position
```

Comment:
The error happened at this import right?

Reply:
Yes